Server/Models/Player.cs

```diff
@@ -17,5 +17,6 @@ public class Player
         public string? PatreonEmblemColor { get; set; }
         public bool? PatreonBootsEnabled { get; set; }
         public List<int>? PatreonChatWheelFavorites { get; set; }
+	    public int Rating { get; set; }
```

Comment:
```suggestion
	    public int Rating12v12 { get; set; }
```

Also, there's a tab at the start of that line
![image](https://user-images.githubusercontent.com/11750369/79050610-96551480-7c44-11ea-9814-3f9cbff4113a.png)